deepchem/feat/molecule_featurizers/mat_featurizer.py

```diff
@@ -28,18 +51,37 @@ class MATFeaturizer(MolecularFeaturizer):
   This class requires RDKit to be installed.
   """
 
-  def __init__(
-      self,
-      one_hot_formal_charge: bool = True,
-  ):
+  def __init__(self):
+    pass
+
+  def construct_mol(self, mol: RDKitMol) -> RDKitMol:
     """
+    Processes an input RDKitMol further to be able to extract id-specific Conformers from it using mol.GetConformer().
+
     Parameters
     ----------
-    one_hot_formal_charge: bool, default True
-      If True, formal charges on atoms are one-hot encoded.
+    mol: RDKitMol
+      RDKit Mol object.
+
+    Returns
+    ----------
+    mol: RDKitMol
+      A processed RDKitMol objeect which is embedded, UFF Optimized and has Hydrogen atoms removed. If the former conditions are not met and there is a value error, then 2D Coordinates are computed instead.
     """
+    try:
+      from rdkit.Chem import AllChem
+      from rdkit import Chem
+    except:
```

Comment:
This is too broad a try/except. We should lilkely only catch a `ModuleNotFound` error

Reply:
Changed :+1: